patches/chrome-android-java-res-layout-bottom_toolbar_browsing.xml.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/chrome/android/java/res/layout/bottom_toolbar_browsing.xml b/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
+index 0a3a1a7c8944c3b9be7fe8d02e90759677b3ad63..89dbfd4be6970542a4c956532458b25684918264 100644
+--- a/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
++++ b/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
+@@ -35,23 +35,7 @@
+ 
+     <include layout="@layout/toolbar_space" />
+ 
+-    <LinearLayout
```

Comment:
please use html comments `<!--` `-->` instead of deleting to reduce the diff

Reply:
done

Reply:
Sorry, should have specified to put them on the line before/after so it's an addition instead of a change. 
```
<--
<LinearLayout
...
</LinearLayout>
-->
```